src/main/java/n/N.java

```diff
@@ -0,0 +1,215 @@
+package n;
+
+import n.exceptions.EmptyTaskDescriptionException;
+import n.exceptions.NoTaskTypeException;
+import n.task.Deadline;
+import n.task.ToDo;
+import n.task.Type;
+import n.task.Task;
+import n.task.Event;
+
+import java.util.Scanner;
+
+public class N {
+    public static final int MAX_LIST_ITEMS = 100;
+    public static final String NO_TASK_TYPE_ERROR = "oHno, you did not specify your task type :O";
+    public static final String NO_TASK_INDEX_ERROR = "Task number not provided, please try again ...";
+    public static final String INVALID_TASK_INDEX_ERROR = "invalid task!";
+    public static final String INPUT_INSTRUCTION = "    Please input in this format:\n";
+    public static final String TODO_INPUT_FORMAT = "    todo [task]";
+    public static final String EVENT_INPUT_FORMAT = "    event [task] /from [start time] /to [end time]";
+    public static final String DEADLINE_INPUT_FORMAT = "    deadline [task] /by [deadline]";
+    public static final String LOGO = " ____   ___\n"
+            + "|    \\ |   |\n"
+            + "|     \\|   |\n"
+            + "|          |\n"
+            + "|   |\\     |\n"
+            + "|___| \\____| .chatbot :)";
+    static Task[] taskList = new Task[MAX_LIST_ITEMS];
+    static int taskCount = 0;
+
+    public static void printLine() {
+        System.out.println("    ____________________________________________________________\n");
+    }
+
+    public static void printMessage(String message) {
+        printLine();
+        System.out.println("    " +message);
+        printLine();
+    }
+
+    public static void printTaskList() {
+        if (taskCount == 0) {
+            printMessage("no tasks added, wake up pleasee!");
+        } else {
+            printLine();
+            System.out.println("    Here are the tasks in your list:");
+            for(int i = 0; i < taskCount; i++) {
+                System.out.println("    " + taskList[i]);
+            }
+            printLine();
+        }
+    }
+
+    public static void changeTaskStatus(int taskIndex, boolean newStatus) {
+        String outputMessage;
+        //check to ensure that task to be marked/unmarked exists in the list
+        if (taskIndex < taskCount) {
+            //when no change in status is required
+            if (taskList[taskIndex].isDone() == newStatus) {
+                //generate output message based on current task status
+                outputMessage = (newStatus) ?
+                        //output message if task has already been marked done
+                        "Task " +taskList[taskIndex].getIndex()+ " is already completed!" :
+                        //output message if task has not been marked done
+                        "Task " +taskList[taskIndex].getIndex()+ " is already unmarked, complete other tasks!";
```

Comment:
consider extracting this into a different function to improve readability and SLAP. 

you can also avoid using the ternary shorthand (?:) operator to improve readability too. 

https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#slap-hard
https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-complicated-expressions